### DISCUSSION TITLE
Has the number of available {stationId}/observations returned changed

### DISCUSSION THREAD

**Discussion by kpm-at-hfi** [weather-gov/api#338] [Q&A] [answered]
Not too long ago I'd been hitting `https://api.weather.gov/stations/${weather_station}/observations` and I think getting over 20 days worth of hourly observations. Hitting this today (it's been a couple weeks since I was monkeying with this) I'm seeing only about the past week's worth of data.

Has this endpoint changed? (I did attempt to find this in the SCNs but didn't see it.)

**Comment by scadergit** [comment, marked as answer]
The API only provides 3 days worth of observations. This was set to replicate the data prevoiusly provided by forecast.weather.gov. Historical observations fall under [Climate services](https://w2.weather.gov/climate/), which has yet to be implemented into the API (mostly due to infrastructure scope).

You are correct it was there a few weeks ago, but that was not intentional and has been corrected.

**Comment by kpm-at-hfi** [comment]
<weeps>

But yeah... makes sense... thanks!
